<!-- Make sure launch ctrl_preapare first -->
<launch>
    <param name="/use_sim_time" value="true" />
    <node pkg="thewalkingdead" type="task3.py" name="task3" output="screen"/>
</launch>